src/main/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/ChoiceListProvider.java

```diff
@@ -104,6 +104,10 @@ public void onBuildCompletedWithValue(
     {
         // Nothing to do.
     }
+
+    public boolean requiresBuildPermission() {
+        return true;
+    }

```

Comment:
Please add a javadoc so that developers creating a class inheriting `ChoiceListProvidrr` can see whather they override this to return `false`.
And also add `@since 1.8.0`.